### ISSUE TITLE
not work in chrome

### ISSUE THREAD

**Issue by wqingzex** [gabrielgodoy-zz/github-stars-manager#5]
```
Uncaught (in promise) TypeError: Cannot read property 'innerHTML' of null
    at s (chrome-extension://epooeampfghdkampjnocabjniefmnkab/dist/content_script.js:6:7132)
    at chrome-extension://epooeampfghdkampjnocabjniefmnkab/dist/content_script.js:4:157
    at a (chrome-extension://epooeampfghdkampjnocabjniefmnkab/dist/content_script.js:5:30490)
    at Generator._invoke (chrome-extension://epooeampfghdkampjnocabjniefmnkab/dist/content_script.js:5:31899)
    at Generator.e.(anonymous function) [as next] (chrome-extension://epooeampfghdkampjnocabjniefmnkab/dist/content_script.js:5:30669)
    at r (chrome-extension://epooeampfghdkampjnocabjniefmnkab/dist/content_script.js:3:30610)
    at chrome-extension://epooeampfghdkampjnocabjniefmnkab/dist/content_script.js:3:30710
    at <anonymous>
```
I think the html of github has changed

**Comment by gabrielgodoy-zz** [comment]
Yes, Github changed some css class.

This is now fixed on version 1.4
https://chrome.google.com/webstore/detail/github-stars-manager/epooeampfghdkampjnocabjniefmnkab

Thanks!

**Comment by cmpute** [comment]
Similar issue here. I've just installed the extension today.
```
content_script.js:6
Uncaught (in promise) TypeError: Cannot read property 'innerHTML' of null
    at l (chrome-extension://epooeampfghdkampjnocabjniefmnkab/dist/content_script.js:6:10679)
    at chrome-extension://epooeampfghdkampjnocabjniefmnkab/dist/content_script.js:6:12214
    at a (chrome-extension://epooeampfghdkampjnocabjniefmnkab/dist/content_script.js:5:30490)
    at Generator._invoke (chrome-extension://epooeampfghdkampjnocabjniefmnkab/dist/content_script.js:5:31899)
    at Generator.e.(anonymous function) [as next] (chrome-extension://epooeampfghdkampjnocabjniefmnkab/dist/content_script.js:5:30669)
    at r (chrome-extension://epooeampfghdkampjnocabjniefmnkab/dist/content_script.js:6:6894)
    at chrome-extension://epooeampfghdkampjnocabjniefmnkab/dist/content_script.js:6:7041
    at Promise (<anonymous>)
    at chrome-extension://epooeampfghdkampjnocabjniefmnkab/dist/content_script.js:6:6838
    at chrome-extension://epooeampfghdkampjnocabjniefmnkab/dist/content_script.js:6:12365
```
```
content_script.js:4
Uncaught (in promise) TypeError: Cannot read property 'querySelectorAll' of null
    at a (chrome-extension://epooeampfghdkampjnocabjniefmnkab/dist/content_script.js:4:8653)
    at chrome-extension://epooeampfghdkampjnocabjniefmnkab/dist/content_script.js:4:333
    at a (chrome-extension://epooeampfghdkampjnocabjniefmnkab/dist/content_script.js:5:30490)
    at Generator._invoke (chrome-extension://epooeampfghdkampjnocabjniefmnkab/dist/content_script.js:5:31899)
    at Generator.e.(anonymous function) [as next] (chrome-extension://epooeampfghdkampjnocabjniefmnkab/dist/content_script.js:5:30669)
    at r (chrome-extension://epooeampfghdkampjnocabjniefmnkab/dist/content_script.js:3:30610)
    at chrome-extension://epooeampfghdkampjnocabjniefmnkab/dist/content_script.js:3:30710
    at <anonymous>
```

**Comment by gabrielgodoy-zz** [comment]
Hey @cmpute , fixed for now on version 1.5

https://chrome.google.com/webstore/detail/github-stars-manager/epooeampfghdkampjnocabjniefmnkab

**Comment by cmpute** [comment]
Thanks! Immediately works~
